src/seedu/addressbook/AddressBook.java

```diff
@@ -607,6 +805,12 @@ private static String getUserInput() {
         }
         return inputLine;
     }
+    private static String getEditUserInput() {
+        System.out.print(LINE_PREFIX + "Enter command: ");
```

Comment:
isn't there a method to show something to the user already? please use that.
notice how using a print statement here reduces the SLAP of this method?